crates/sui-graphql-rpc/src/server/graphiql_server.rs

```diff
@@ -18,19 +18,28 @@ async fn graphiql(ide_title: axum::Extension<Option<String>>) -> impl axum::resp
 
 pub async fn start_graphiql_server(server_config: &ServerConfig) -> Result<(), Error> {
     info!("Starting server with config: {:?}", server_config);
-    start_graphiql_server_impl(ServerBuilder::from_config(server_config).await?).await
+    start_graphiql_server_impl(
+        ServerBuilder::from_config(server_config).await?,
+        server_config.clone(),
```

Comment:
It seems like this is only used for the IDE title, could we just pass that down?